### DISCUSSION TITLE
Can I configure which topics are displayed on the intertopic distance map?

### DISCUSSION THREAD

**Discussion by hashtagnophilter** [MaartenGr/BERTopic#1101] [Q&A] [answered]
Once I have generated my model, how can I manually specify which topics are displayed on the intertopic distance map?

**Comment by MaartenGr** [comment, marked as answer]
When you run `topic_model.visualize_topics()` there is a `topics` variable that allows you to select the relevant topics. I would advise reading through the documentation [here](https://maartengr.github.io/BERTopic/api/plotting/topics.html) for more information.

**Reply by hashtagnophilter** [reply]
Thanks @MaartenGr for the response - apologies I have been away. Missed this in the documentation. Loving the recent updates
